### ISSUE TITLE
String Thickness

### ISSUE THREAD

**Issue by Aduffy** [moonwave99/fretboard.js#9]
An option for string thickness such as a true/false for "scaleStrings" would be nice.

**Comment by moonwave99** [comment]
I didn't get it - are `stringWidth` and  `fretWidth` options not enough?

```js
const f = new Fretboard({
  stringWidth: 3,
  fretWidth: 3
});

```

**Comment by Aduffy** [comment]
Sorry I was not clear.. thinking something like this 

![image](https://user-images.githubusercontent.com/265857/100283353-36d11e00-2f22-11eb-98eb-0c0f067fad3e.png)

**Comment by moonwave99** [comment]
Ah I see!

Ready in 92cccf408b0dab0714ab18b1af26e794633eb66f, will be part of next release ^^

Example:

```js
const fretboard = new Fretboard({
  stringWidth: [.75, 1, 1.5, 2, 2.5, 3]
});
```

**Comment by Aduffy** [comment]
Sweet! That was fast!!!

**Comment by moonwave99** [comment]
Fixed in v0.1.9
